plext/UsingProceduralLanguages.html.md.erb

```diff
@@ -1,13 +1,16 @@
 ---
-title: Using Procedural Languages and Extensions in HAWQ
+title: Using Languages and Extensions in HAWQ
 ---
 
-HAWQ allows user-defined functions to be written in other languages besides SQL and C. These other languages are generically called *procedural languages* (PLs).
+HAWQ supports user-defined functions created with the SQL and C built-in languages, including supporting user-defined aliases for internal functions.
```

Comment:
This needs a bit of an edit:  HAWQ supports user-defined functions **that are** created with the SQL and C built-in languages, **and also supports** user-defined aliases for internal functions.